mitiq/calibration/calibrator.py

```diff
@@ -21,16 +21,25 @@
     ZNESettings,
     Strategy,
     BenchmarkProblem,
+    MitigationTechnique,
 )
 from mitiq.interface import convert_from_mitiq
 
-TABLE_HEADER_STR = (
+TABLE_HEADER_STR_ZNE = (
```

Comment:
I think the method should come first in the variable name now that there are two headers. So something like `ZNE_TABLE_HEADER`, WDYT?

Reply:
Sounds good, I've updated the names accordingly.